client/components/TestRenderer/OutputTextArea/index.jsx

```diff
@@ -78,6 +81,7 @@ const OutputTextArea = ({ commandIndex, atOutput, isSubmitted }) => {
                 value={atOutput.value}
                 onChange={e => atOutput.change(e.target.value)}
                 disabled={noOutput}
+                readOnly={readOnly}
             />
```

Comment:
I agree the clipping is an issue and it would be a good idea to raise the height of that textarea by 0.5em just to show that the scroll is available because some OS's don't even show the scrollbar when it isn't focused.

RE stripping out the new lines, my sense so far based on previous conversations is that we don't want to do any manipulation of the output since information that we deem not useful may actually be useful for tracking changing behavior over time.

Reply:
I ended up raising the default size of the text area by 1/2 line to make the scroll obvious at least. I think any manipulation of output would fall beyond the scope of this PR.

Reply:
I'm cool with that! Thanks!